src/test/java/seedu/address/model/session/SessionDateTest.java

```diff
@@ -24,55 +23,54 @@ public void isValidSessionDate() {
         String correctTimeValue = "23:59";
 
         // checks for empty input
-        assertThrows(SessionException.class, () -> new SessionDate("", correctTimeValue));
+        assertFalse(SessionDate.isValidSessionDate("", correctTimeValue));
 
         // checks for invalid month
-        assertThrows(SessionException.class, () -> new SessionDate("2021-13-01", correctTimeValue));
-        assertThrows(SessionException.class, () -> new SessionDate("2021-00-01", correctTimeValue));
+        assertFalse(SessionDate.isValidSessionDate("2021-13-01", correctTimeValue));
+        assertFalse(SessionDate.isValidSessionDate("2021-00-01", correctTimeValue));
 
         // checks for invalid day
```

Comment:
Perhaps we could include an assertion for invalid days in month, for example 31 Apr is _invalid_ since April only has 30 days, something similar to leap year check! (does the exception catch this actually?)

Reply:
Good suggestion! The isValidSessionDate() should assertFalse correctly to this test case.